packages/Card/Card.jsx

```diff
@@ -71,9 +74,37 @@ const Card = ({ variant, children, fullHeight, spacing, ...rest }) => {
     spacingProps.inset = 4
   }
 
+  const ContainerStyles = imageProps =>
+    imageProps &&
+    imageProps.position &&
+    handleResponsiveStyles({ position: imageProps.position }, ({ position }) => {
+      if (!imageProps) return {}
+      const direction = {
+        left: 'row',
+        right: 'row-reverse',
+        top: 'column',
+        bottom: 'column-reverse',
+        none: 'row',
+      }
+      const styles = {
+        display: 'flex',
+        flexDirection: direction[position],
+        justifyContent: 'space-between',
+        '> img': {
+          display: position === 'none' ? 'none' : 'block',
+          margin: 'auto',
+        },
+      }
+      return styles
+    })
+
+  const StyledDiv = styled.div(ContainerStyles)
   return (
-    <StyledCard {...safeRest(rest)} fullHeight={fullHeight} variant={variant} {...spacingProps}>
-      {children}
+    <StyledCard {...safeRest(rest)} fullHeight={fullHeight} variant={variant}>
+      <StyledDiv {...fullBleedImage}>
+        {fullBleedImage && <Image {...fullBleedImage} />}
```

Comment:
Can you please pass individual props instead of prop spreading?

There are two reasons I can think of:
1- Readability: When I just see `{...fullBleedImage}` I have no idea what exactly is being passed to `Image`.
2- Unnecessary prop passing: `Image` component by design doesn't take a `position` prop when you spread the `fullBleedImage` object, you end up passing an unnecessary position prop to it.

Reply:
Makes sense, change implemented :)